csm/conf/etc/csm/csm.conf

```diff
@@ -188,4 +188,9 @@ OPENLDAP:
 CLUSTER_ADMIN:
     user: "cortxadmin"
     secret: "Cortxadmin@123"
-    email: "cortxadmin@seagate.com"
\ No newline at end of file
+    email: "cortxadmin@seagate.com"
+
+STORAGE_CAPACITY_MANAGMENT:
+    auth: "disable"
+    hctl_service_endpoint: ""
+    cluster_status_api: "/cluster/status"
```

Comment:
Add support for V1 for this API. Ticket - https://jts.seagate.com/browse/EOS-27313

Reply:
Modifications done